docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -0,0 +1,216 @@
+--
+
+. On the systems where other {agent}s are running, add:
++
+--
+* (Optional) The CA certificate to use to connect to {es}. This is optional
+because you can embed this certificate later when you specify {fleet}
+settings.
+* The CA certificate to use to connect to {fleet-server}. This is the CA you
+used to <<generate-fleet-server-certs,generate a certificate and key>> for
+{fleet-server}. For example, `ca.crt`.
+--
+
+. Configure {fleet} settings:
+
+.. In {kib} go to *Management > {fleet}*.
+
+.. In the upper-right corner, click *Fleet settings*.
+
+.. In the *Fleet Server hosts* field, specify the
+URLs {agent}s will use to connect to {fleet-server}. For example,
+https://192.0.2.1:8220, where 192.0.2.1 is the host IP where you will install
+{fleet-server}.
+
+.. In the *Elasticsearch host* field, specify the {es} URLs where {agent}s will
+send data. For example, https://192.0.2.0:9200.
++
+Use the `https` protocol for both host settings.
+
+.. Under *Elasticsearch output configuration (YAML)*, set
+`ssl.certificate_authorities` and specify the CA certificate to use to connect
+to {es}. You can specify a list of file paths (if the files are available), or
+embed a certificate directly in the YAML configuration:
++
+--
+Path example:
+
+[source,yaml]
+----
+ssl.certificate_authorities: ["/home/vagrant/certs/elasticsearch-ca.crt"] <1>
+----
+<1> The path to the CA certificate on the {agent} host.
+
+Pasted certificate example:
+
+[source,yaml]
+----
+ssl:
+  certificate_authorities:
+  - |
+    -----BEGIN CERTIFICATE-----
+    MIIDSjCCAjKgAwIBAgIVAKlphSqJclcni3P83gVsirxzuDuwMA0GCSqGSIb3DQEB
+    CwUAMDQxMjAwBgNVBAMTKUVsYXN0aWMgQ2VydGlmaWNhdGUgVG9vbCBBdXRvZ2Vu
+    ZXJhdGVkIENBMB4XDTIxMDYxNzAxMzIyOVoXDTI0MDYxNjAxMzIyOVowNDEyMDAG
+    A1UEAxMpRWxhc3RpYyBDZXJ0aWZpY2F0ZSBUb29sIEF1dG9nZW5lcmF0ZWQgQ0Ew
+    ggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDOFgtVri7Msy2iR33nLrVO
+    /M/6IyF72kFXup1E67TzetI22avOxNlq+HZTpZoWGV1I4RgxiQeN12FLuxxhd9nm
+    rxfZEqpuIjvo6fvU9ifC03WjXg1opgdEb6JqH93RHKw0PYimxhQfFcwrKxFseHUx
+    DeUNQgHkMQhDZgIfNgr9H/1X6qSU4h4LemyobKY3HDKY6pGsuBzsF4iOCtIitE9p
+    sagiWR21l1gW/lNaEW2ICKhJXbaqbE/pis45/yyPI4Q1Jd1VqZv744ejnZJnpAx9
+    mYSE5RqssMeV6Wlmu1xWljOPeerOVIKUfHY38y8GZwk7TNYAMajratG2dj+v9eAV
+    AgMBAAGjUzBRMB0GA1UdDgQWBBSCNCjkb66eVsIaa+AouwUsxU4b6zAfBgNVHSME
+    GDAWgBSCNCjkb66eVsIaa+AouwUsxU4b6zAPBgNVHRMBAf8EBTADAQH/MA0GCSqG
+    SIb3DQEBCwUAA4IBAQBVSbRObxPwYFk0nqF+THQDG/JfpAP/R6g+tagFIBkATLTu
+    zeZ6oJggWNSfgcBviTpXc6i1AT3V3iqzq9KZ5rfm9ckeJmjBd9gAcyqaeF/YpWEb
+    ZAtbxfgPLI3jK+Sn8S9fI/4djEUl6F/kARpq5ljYHt9BKlBDyL2sHymQcrDC3pTZ
+    hEOM4cDbyKHgt/rjcNhPRn/q8g3dDhBdzjlNzaCNH/kmqWpot9AwmhhfPTcf1VRc
+    gxdg0CTQvQvuceEvIYYYVGh/cIsIhV2AyiNBzV5jJw5ztQoVyWvdqn3B1YpMP8oK
+    +nadUcactH4gbsX+oXRULNC7Cdd9bp2G7sQc+aZm
+    -----END CERTIFICATE-----
+----
+--
+
+. Configure the certificates for {fleet-server}:
+
+.. If you don't already have a {fleet-server} service token, click *Fleet* and
```

Comment:
Should we show the API call for this too?

Reply:
Yes, I would say so.  A screenshot of where this actually is in Kibana would be helpful too.

Reply:
I'll add some screenshots when I do final verification of the docs. We try to keep screen captures to a minimum because they get stale quickly, but I'll add the important ones.

Reply:
```suggestion
.. If you don't already have a {fleet-server} service token, click *Add agent* > *Enroll in Fleet* > *Generate service token* and
```